model/draft.go

```diff
@@ -23,6 +23,7 @@ type Draft struct {
 	Props    StringInterface `json:"props"` // Deprecated: use GetProps()
 	FileIds  StringArray     `json:"file_ids,omitempty"`
 	Metadata *PostMetadata   `json:"metadata,omitempty"`
+	Priority StringInterface `json:"priority,omitempty"`
```

Comment:
Just curious here, could we have used `PostPriority` struct type here instead of `StringInterface`. And in case of `StringInterface` doesn't it need a maxSizeLimit check in the `IsValid` method, similar to `Props`?

Reply:
I tried to, but I was having errors which I didn't understand fully how to fix.
I'll add the `IsValid` yes, thanks